server/vulnerabilities/nvd/cve_test.go

```diff
@@ -424,6 +463,18 @@ func TestGetMatchingVersionEndExcluding(t *testing.T) {
 			want:    "",
 			wantErr: false,
 		},
+		{
+			name: "Version exceeds version end excluding",
+			cve:  "CVE-2022-4610",
+			meta: &wfn.Attributes{
+				Vendor:   "clickstudios",
+				Product:  "passwordstate",
+				Version:  "9.5.8.4",
```

Comment:
Curious, does our version comparison work for these cases where version has 4 parts (4 numbers)? Go's `semver` package expects 3 parts, like major.minor.patch-build

Reply:
we attempt to convert non-semantic to semantic.  In this case that string would be converted to `9.5.8+4`.  It's not full proof, so I'm sure there's some more edge cases out there we'll need to cover.